src/mozanalysis/experiment.py

```diff
@@ -313,29 +341,35 @@ def build_query(
         )
 
         return """
-    WITH analysis_windows AS (
-        {analysis_windows_query}
-    ),
-    raw_enrollments AS ({enrollments_query}),
-    segmented_enrollments AS ({segments_query}),
-    enrollments AS (
+    CREATE TEMPORARY TABLE enrollments AS (
```

Comment:
Can you remind me how temporary `TEMPORARY` is? Might multiple simultaneous instances of mozanalysis clobber each other? Might sequentially running jobs hit issues?

Reply:
Extremely; they're not accessible except from the script that creates them. See https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#temporary_tables